thunder.post_update.php

```diff
@@ -8,10 +8,12 @@
 /**
  * Install the Configuration Selector module.
  */
-function thunder_post_update_install_config_selector() {
+function thunder_post_update_ensure_config_selector_installed() {
   /** @var \Drupal\thunder_updater\Updater $thunderUpdater */
   $thunderUpdater = \Drupal::service('thunder_updater');
-  $thunderUpdater->installModules(['thunder_post_update_install_config_selector' => 'config_selector']);
+  if (!\Drupal::moduleHandler()->moduleExists('config_selector')) {
+    $thunderUpdater->installModules(['thunder_post_update_install_config_selector' => 'config_selector']);
```

Comment:
I think keeping the same name here makes sense - this means there should only be one checkbox about installing this module.

Reply:
Perhaps a comment might be useful for our futureselves to understand why.

Reply:
That said this update is going to be a part of 2.15 so maybe we will need a new entry in docroot/profiles/contrib/thunder/modules/thunder_updater/updates.yml - not sure.

Reply:
I think we don't need a new checkbox entry. For most of the sites this update hook will do nothing. They will not uninstall the module immediately

Reply:
Okay so let's add a comment about why this doesn't match.